packages/router-generator/src/generator.ts

```diff
@@ -280,8 +283,7 @@ export async function generator(config: Config) {
   const routeConfigChildrenText = await buildRouteConfig(routeTree)
 
   const sortedRouteNodes = multiSortBy(routeNodes, [
-    (d) =>
-      d.routePath?.includes(`/${routePathIdPrefix + rootPathId}`) ? -1 : 1,
+    (d) => (d.routePath?.includes(`/${rootPathId}`) ? -1 : 1),
```

Comment:
Same as other check - since we've stripped out the prefix in the getRouteNodes function we don't have to include the routeFilePrefix when filtering the rootPath